android/sdkVersions.json

```diff
@@ -1 +1 @@
-{"sdkVersions":["33.0.0","34.0.0","35.0.0"]}
\ No newline at end of file
+{"sdkVersions":["34.0.0","35.0.0"]}
```

Comment:
That's interesting that this file does not contain `36.0.0` that we're supporting 🤔 cc @tsapeta

Reply:
🤦‍♂ 
Could you @lukmccall check whether it's being updated by `et add-sdk`?